AllCommandsOption.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Huttopia\ConsoleBundle;
+
+class AllCommandsOption
+{
+    public static function parseAllCommandsOption(array &$argv): bool
+    {
+        $allCommandsKey = array_search('--all-commands', $argv);
+        if (is_int($allCommandsKey) || is_string($allCommandsKey)) {
+            unset($argv[$allCommandsKey]);
+            unset($_SERVER['argv'][$allCommandsKey]);
```

Comment:
Symfony use `$_SERVER['argv']` and not `$argv`. To be "perfect" i prefer removing in `$argv` and `$_SERVER['argv']`.